autosklearn/evaluation/__init__.py

```diff
@@ -39,7 +39,27 @@ def fit_predict_try_except_decorator(ta, queue, cost_for_crash, **kwargs):
                    'additional_run_info': {'traceback': exception_traceback,
                                            'error': error_message},
                    'status': StatusType.CRASHED,
-                   'final_queue_element': True})
+                   'final_queue_element': True}, block=True)
```

Comment:
One question, block=True will wait forever right? That is, if something goes south, autosklearn will get stuck here without printing the below message?

Reply:
Thanks for spotting this. I re-organized the order, do you think it's fine like this?